packages/viewer/src/components.d.ts

```diff
@@ -41,7 +41,7 @@ export namespace Components {
           * Internal API.
           * @private
          */
-        "registerCommand": <R, T>(id: string, factory: CommandFactory<R>, thisArg?: T) => Promise<Disposable>;
+        "registerCommand": <R, T>(id: string, factory: CommandFactory<R>, thisArg?: T | undefined) => Promise<Disposable>;
```

Comment:
was this a result of the config change?

Reply:
yes it was, It is automatically updating for me here.